lib/v2/screens/dashboard/components/currency_info_card_widget.dart

```diff
@@ -1,82 +1,67 @@
-                            height: 42,
-                            decoration: BoxDecoration(
-                                borderRadius: BorderRadius.circular(8),
-                                image: DecorationImage(image: AssetImage(logo), fit: BoxFit.fill))),
-                      ],
-                    ),
-                    const SizedBox(
-                      height: 50,
-                    ),
-                    Text("Balance", style: Theme.of(context).textTheme.subtitle2!.copyWith(color: textColor)),
-                    const SizedBox(
-                      height: 6,
-                    ),
-                    Text(state.cardDisplayText,
-                        style: Theme.of(context).textTheme.headline5!.copyWith(color: textColor)),
-                    const SizedBox(
-                      height: 6,
-                    ),
-                    Text("", style: Theme.of(context).textTheme.subtitle2!.copyWith(color: textColor))
-                  ],
-                ),
-              )
-            ]));
-      }),
-    );
+      return Container(
+          width: cardWidth,
+          height: cardHeight,
+          decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(24),
+              image: DecorationImage(image: AssetImage(tokenBalance.token.backgroundImage), fit: BoxFit.fill)),
+          child: Stack(children: <Widget>[
+            Padding(
+              padding: const EdgeInsets.only(top: 20, right: 20, left: 20),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Expanded(
+                          child: Text(
+                        tokenBalance.token.name,
+                        style: Theme.of(context).textTheme.headline7.copyWith(color: textColor),
+                      )),
+                      Container(
+                          width: 42,
+                          height: 42,
+                          decoration: BoxDecoration(
+                              borderRadius: BorderRadius.circular(8),
+                              image: DecorationImage(image: AssetImage(tokenBalance.token.logo), fit: BoxFit.fill))),
+                    ],
+                  ),
+                  const SizedBox(
+                    height: 50,
+                  ),
+                  Text("Balance", style: Theme.of(context).textTheme.subtitle2!.copyWith(color: textColor)),
+                  const SizedBox(
+                    height: 6,
+                  ),
+                  Text(tokenBalance.displayQuantity,
+                      style: Theme.of(context).textTheme.headline5!.copyWith(color: textColor)),
+                  const SizedBox(
+                    height: 6,
+                  ),
+                  Text("", style: Theme.of(context).textTheme.subtitle2!.copyWith(color: textColor))
```

Comment:
What is this empty text used for?

Reply:
Hmm I guess there used to be a subtitle to the card titles...